src/Raven.Server/Documents/Indexes/Persistence/Corax/CoraxQueryBuilder.cs

```diff
@@ -287,12 +287,23 @@ IQueryMatch HandleStringUnaryMatch()
 
             var fieldId = QueryBuilderHelper.GetFieldId(fieldName, index, indexMapping, queryMapping, exact);
 
+            var hasGotTheRealType = false;
```

Comment:
Please fix the warning `The variable 'hasGotTheRealType' is assigned but its value is never used`